stable/atlantis/templates/statefulset.yaml

```diff
@@ -225,7 +229,7 @@ spec:
     spec:
       accessModes: ["ReadWriteOnce"] # Volume should not be shared by multiple nodes.
       {{- if .Values.atlantis_data_storageClass }}
-      storageClass: {{ .Values.atlantis_data_storageClass }} # Storage class of the volume
+      storageClassName: {{ .Values.atlantis_data_storageClass }} # Storage class of the volume
```

Comment:
@sstarcher what do you think about renaming this: `.Values.atlantis_data_storageClass` -> `.Values.storageClassName`